pkg/storaged/details.js

```diff
@@ -34,7 +34,25 @@ define([
      */
 
     function init_details(client, jobs) {
-        var type, name;
+        var type, name, state;
+
+        function ensure_state_multipathd_running() {
+            if (state.multipathd_running === undefined) {
```

Comment:
Yes, agreed, why not be perfect. :-) This needs non-trivial code, however, and we should put that into the systemd package.  See the comments at the top of pkg/systemd/init.js for why this isn't entirely trivial.